src/transformers/models/udop/tokenization_udop_fast.py

```diff
@@ -0,0 +1,1040 @@
+        only_label_first_subword (`bool`, *optional*, defaults to `True`):
+            Whether or not to only label the first subword, in case word labels are provided.
+        extra_ids (`int`, *optional*, defaults to 100):
+            The number of extra IDs added to the special tokens.
+        loc_extra_ids (`int`, *optional*, defaults to 501):
+            The number of extra location IDs added to the special tokens.
+        other_extra_ids (`int`, *optional*, defaults to 200):
+            The number of extra other IDs added to the special tokens.
+        additional_special_tokens (`List[str]`, *optional*, defaults to `["<s>NOTUSED", "</s>NOTUSED"]`):
+            Additional special tokens used by the tokenizer.
+    """
+
+    vocab_files_names = VOCAB_FILES_NAMES
+    pretrained_vocab_files_map = PRETRAINED_VOCAB_FILES_MAP
+    max_model_input_sizes = PRETRAINED_POSITIONAL_EMBEDDINGS_SIZES
+    model_input_names = ["input_ids", "attention_mask"]
+    slow_tokenizer_class = UdopTokenizer
+
+    def __init__(
+        self,
+        vocab_file=None,
+        tokenizer_file=None,
+        eos_token="</s>",
+        sep_token="</s>",
+        unk_token="<unk>",
+        pad_token="<pad>",
+        sep_token_box=[1000, 1000, 1000, 1000],
+        pad_token_box=[0, 0, 0, 0],
+        pad_token_label=-100,
+        only_label_first_subword=True,
+        extra_ids=100,
+        loc_extra_ids=501,
+        other_extra_ids=200,
+        additional_special_tokens=None,
+        **kwargs,
+    ):
+        # Add extra_ids to the special token list
+        if extra_ids > 0 and (additional_special_tokens is None or "<extra_id_0>" not in additional_special_tokens):
+            if additional_special_tokens is None:
+                additional_special_tokens = ["<extra_id_{}>".format(i) for i in range(extra_ids - 1, -1, -1)]
+            additional_special_tokens.extend(["<extra_l_id_{}>".format(i) for i in range(extra_ids - 1, -1, -1)])
+            additional_special_tokens.extend(["</extra_l_id_{}>".format(i) for i in range(extra_ids - 1, -1, -1)])
+            additional_special_tokens.extend(["<extra_t_id_{}>".format(i) for i in range(extra_ids - 1, -1, -1)])
+            additional_special_tokens.extend(["</extra_t_id_{}>".format(i) for i in range(extra_ids - 1, -1, -1)])
+
+        if loc_extra_ids > 0 and (additional_special_tokens is None or "<loc_0>" not in additional_special_tokens):
+            if additional_special_tokens is None:
+                additional_special_tokens = []
+            additional_special_tokens.extend(["<loc_{}>".format(i) for i in range(loc_extra_ids - 1, -1, -1)])
+
+        if other_extra_ids > 0 and (additional_special_tokens is None or "<other_0>" not in additional_special_tokens):
+            if additional_special_tokens is None:
+                additional_special_tokens = []
+            additional_special_tokens.extend(["<other_{}>".format(i) for i in range(other_extra_ids - 1, -1, -1)])
```

Comment:
```suggestion
```
add this to the conversion script rather

Reply:
Oh so you're saying I should add these tokens in the conversion script?